test/tick_multi_cluster_test.go

```diff
@@ -175,3 +192,50 @@ func TestTickMultiCluster(t *testing.T) {
 		})
 	}
 }
+
+func buildAllAmis(t *testing.T, awsRegion string, telegrafPackerInfo *PackerInfo, influxdbPackerInfo *PackerInfo, chronografPackerInfo *PackerInfo, kapacitorPackerInfo *PackerInfo, examplesDir string) *InfluxAmis {
```

Comment:
You can ignore this for now, but for future reference, terratest has a function that builds multiple amis at once: 

https://github.com/gruntwork-io/terratest/blob/master/modules/packer/packer.go#L50